Gems/Atom/RHI/DX12/Code/Source/Platform/Windows/RHI/Device_Windows.cpp

```diff
@@ -310,12 +310,15 @@ namespace AZ
                 return "D3D12_DRED_ALLOCATION_TYPE_VIDEO_MOTION_VECTOR_HEAP";
             case D3D12_DRED_ALLOCATION_TYPE_VIDEO_EXTENSION_COMMAND:
                 return "D3D12_DRED_ALLOCATION_TYPE_VIDEO_EXTENSION_COMMAND";
-#ifdef __ID3D12DeviceRemovedExtendedDataSettings1_INTERFACE_DEFINED__
-            case D3D12_DRED_ALLOCATION_TYPE_VIDEO_ENCODER:
+
+            // NOTE: These enums are not defined in older Win10 SDKs
```

Comment:
You can specify the exact version and up in the comment: `Windows SDK 10 (10.0.203.348)`

Reply:
I haven't actually confirmed it just yet but agreed

Reply:
I updated the comment